manifests/course/get_hiera2.pp

```diff
@@ -6,4 +6,8 @@
     recurse => true,
     source  => "puppet:///modules/course_selector/get_hiera/${unit}/hieradata",
   }
+  file {'/opt/puppetlabs/facter/facts.d/my_facts.txt':
```

Comment:
We tend to tell people that stuff in `/opt/puppetlabs` is for vendor stuff and local stuff goes in `/etc/puppetlabs`